grpc.go

```diff
@@ -46,7 +46,7 @@ type GRPCLogFormatter interface {
 
 type grpcConfig struct {
 	ignoredMethods map[string]struct{}
-	metadata       map[string]struct{}
+	metadata       map[string]string
```

Comment:
> colon is used for dividing header name and alias key. If colon is not exist, use header name as a key. 

I suggested colon separated string for alias. do you think alias map is better?

Reply:
I was mistaken. It's corrected.